sql/core/src/test/resources/sql-tests/results/datetime-legacy.sql.out

```diff
@@ -996,3 +996,72 @@ select date_format('2020-01-01', 'YYYY-MM-dd uuuu')
 struct<date_format(CAST(2020-01-01 AS TIMESTAMP), YYYY-MM-dd uuuu):string>
 -- !query output
 2020-01-01 0003
+
+
+-- !query
+create temporary view ttt as select t from VALUES
+ (timestamp '1582-06-01 11:33:33.123UTC+080000'),
+ (timestamp '1970-01-01 00:00:00.000Europe/Paris'),
+ (timestamp '1970-12-31 23:59:59.999Asia/Srednekolymsk'),
+ (timestamp '1996-04-01 00:33:33.123Australia/Darwin'),
+ (timestamp '2018-11-17 13:33:33.123Z'),
+ (timestamp '2020-01-01 01:33:33.123Asia/Shanghai'),
+ (timestamp '2100-01-01 01:33:33.123America/Los_Angeles') tt(t)
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+select date_format(t, 'Y-w-u YYYY-ww-uu YYY-W-uuu YY YYYYY uuuu E EE EEE EEEE') from ttt
+-- !query schema
+struct<date_format(t, Y-w-u YYYY-ww-uu YYY-W-uuu YY YYYYY uuuu E EE EEE EEEE):string>
+-- !query output
+1582-22-4 1582-22-04 1582-5-004 82 01582 0004 Thu Thu Thu Thursday
+1970-1-3 1970-01-03 1970-5-003 70 01970 0003 Wed Wed Wed Wednesday
+1970-53-4 1970-53-04 1970-5-004 70 01970 0004 Thu Thu Thu Thursday
+1996-13-7 1996-13-07 1996-4-007 96 01996 0007 Sun Sun Sun Sunday
+2018-46-6 2018-46-06 2018-3-006 18 02018 0006 Sat Sat Sat Saturday
+2020-1-2 2020-01-02 2020-5-002 20 02020 0002 Tue Tue Tue Tuesday
+2099-53-5 2099-53-05 2099-0-005 99 02099 0005 Fri Fri Fri Friday
+
+
+-- !query
+select date_format(t, 'q qq Q QQ QQQ QQQQ') from ttt
+-- !query schema
+struct<>
+-- !query output
+java.lang.IllegalArgumentException
+Illegal pattern character 'q'
+
+
+-- !query
+select date_format(t, 'y-M-d H:m:s yyyy-MM-dd HH:mm:ss.SSS yy yyy yyyyy MMM MMMM L LL F h hh k kk K KK a') from ttt
+-- !query schema
+struct<date_format(t, y-M-d H:m:s yyyy-MM-dd HH:mm:ss.SSS yy yyy yyyyy MMM MMMM L LL F h hh k kk K KK a):string>
+-- !query output
+1582-5-31 19:40:35 1582-05-31 19:40:35.123 82 1582 01582 May May 5 05 5 7 07 19 19 7 07 PM
+1969-12-31 15:0:0 1969-12-31 15:00:00.000 69 1969 01969 Dec December 12 12 5 3 03 15 15 3 03 PM
+1970-12-31 4:59:59 1970-12-31 04:59:59.999 70 1970 01970 Dec December 12 12 5 4 04 4 04 4 04 AM
+1996-3-31 7:3:33 1996-03-31 07:03:33.123 96 1996 01996 Mar March 3 03 5 7 07 7 07 7 07 AM
+2018-11-17 5:33:33 2018-11-17 05:33:33.123 18 2018 02018 Nov November 11 11 3 5 05 5 05 5 05 AM
+2019-12-31 9:33:33 2019-12-31 09:33:33.123 19 2019 02019 Dec December 12 12 5 9 09 9 09 9 09 AM
+2100-1-1 1:33:33 2100-01-01 01:33:33.123 00 2100 02100 Jan January 1 01 1 1 01 1 01 1 01 AM
+
+
+-- !query
+select date_format(t, 'VV z zz zzz zzzz O OOOO X XX XXX XXXX XXXXX x xx xxx xxxx xxxx xxxxx Z ZZ ZZZ ZZZZ ZZZZZ') from ttt
```

Comment:
can we test `VV` separately? otherwise all other patterns listed here can't be tested with the legacy formatter.